core/5.0/global.d.ts

```diff
@@ -0,0 +1,215 @@
+// Based on https://www.lua.org/manual/5.3/manual.html#6.1
+
+/// <reference lib="es2015.iterable" />
```

Comment:
It's in there because it was part of the original global.d.ts file, too. I also don't believe it is used. Fixed with https://github.com/TypeScriptToLua/lua-types/pull/59/commits/5b379fef0be91d8185ab29d47eb5e4eca3f73f07.